packages/@expo/cli/src/run/ios/runIosAsync.ts

```diff
@@ -32,13 +34,30 @@ export async function runIosAsync(projectRoot: string, options: Options) {
 
   const install = !!options.install;
 
+  let maybePromptToSyncPods = false;
+  // Only prompt to sync pods if the user is not using a custom binary.
   if ((await ensureNativeProjectAsync(projectRoot, { platform: 'ios', install })) && install) {
-    await maybePromptToSyncPodsAsync(projectRoot);
+    maybePromptToSyncPods = true;
```

Comment:
given that this could impact the fingerprint, maybe we will need to run pod install before generating it?

Reply:
this does raise an interesting question to follow up with later though - how should we integrate usage of this build caching with projects that don't use `npx expo run:ios` to build locally? right now they would use `eas build:dev`

Reply:
reverted